integrations/pom.xml

```diff
@@ -79,7 +79,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>mailer</artifactId>
-      <version>1.25</version>
+      <version>1.31-rc320.c4a8e56c2341</version>
```

Comment:
To be updated once https://github.com/jenkinsci/mailer-plugin/pull/80 is merged.

Reply:
```suggestion
      <version>1.32</version>
```